resource-managers/kubernetes/integration-tests/pom.xml

```diff
@@ -102,6 +103,16 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.22.0</version>
```

Comment:
I think we declare the version in the parent POM? (or should). OK to update it while you're at it to the latest version.